ceci/stage.py

```diff
@@ -661,6 +665,16 @@ def execute(cls, args, comm=None):
         if args.memmon:  # pragma: no cover
             monitor = MemoryMonitor.start_in_thread(interval=args.memmon)
 
+        # Now we try to see if the validation step has been changed,
+        # if it has then we will run the validation step, and raise any errors
+        try:
+            stage.validate()
+        except Exception as error:
+            print(f"Looks like there is an validation error in: {cls.name}")
```

Comment:
Maybe slightly more descriptive here as well - if people don't know what the validation steps do then this will confuse them. e.g. "The `validate` method in stage {cls.name} raised an error. This means that checks done on input data before running did not pass". Or something more eloquent.

Reply:
hmm good point, So I looked at this again, and noticed as is it will print this line to the log file associated with the stage, rather than to the cmd line. I figure we would want this message to be printed in the cmd line, and then people can look at the log for why it failed?